cluster-autoscaler/cloudprovider/aws/auto_scaling_groups.go

```diff
@@ -44,6 +44,16 @@ type asgCache struct {
 	explicitlyConfigured  map[AwsRef]bool
 }
 
+type launchTemplate struct {
+	launchTemplateName    string
+	launchTemplateVersion string
+}
+
+type mixedInstancesPolicy struct {
+	launchTemplate *launchTemplate
+	instanceTypes  []string
```

Comment:
recommend calling this field `overrides` or `instanceTypeOverrides` to better describe what they represent, which is the `MixedInstancePolicy.Overrides` field in the ASG.